client/plasmacli/spend.go

```diff
@@ -45,38 +36,35 @@ func init() {
 
 var spendCmd = &cobra.Command{
 	Use:   "spend",
-	Short: "Build, Sign, and Send transactions",
+	Short: "Send a transaction spending utxos",
 	RunE: func(cmd *cobra.Command, args []string) error {
 		ctx := context.NewCLIContext()
+		name := viper.GetString(flagAccount)
```

Comment:
Looks like can only send from one address at a time in this implementation. Should be able to send `--names acc1,acc2` to enable multiple inputs